modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/segment_edit/SegmentEdit.es.js

```diff
@@ -176,40 +188,75 @@ class SegmentEdit extends Component {
 		);
 	};
 
+	_renderLocalizedInputs = () => {
+		const {
+			defaultLanguageId,
+			portletNamespace,
+			values
+		} = this.props;
+
+		const langs = Object.entries(values.name);
+
+		return (
+			langs.map(
+				([key, value]) => {
+					let render;
+					if (key === defaultLanguageId) {
+						render = (<React.Fragment key={key}>
```

Comment:
The `(` should be on its own line. Formatting for multi-line jsx should look something like

```jsx
returnVal = (
     <SomeComponent>
            <div />
     </SomeComponent>
);